packages/cody/src/lib/hooks/on-document.ts

```diff
@@ -102,6 +102,15 @@ export const onDocument: CodyHook<Context> = async (vo: Node, ctx: Context) => {
 
       withErrorCheck(registerQueryResolver, [service, vo, ctx, tree]);
 
+      generateFiles(tree, __dirname + '/query-files/fe', ctx.feSrc, {
+        tmpl: "",
+        service: serviceNames.fileName,
+        serviceNames,
```

Comment:
I'm reusing existing variables for the substitutions object here, because the cody hook `on-document` already contains code generation for backend and @app/shared packages. `serviceNames` for example is an object returned from a helper function `names(inputName: string)` that converts the inputName to:

```
{
  fileName: "input-name",
  className: "InputName",
  propertyName: "inputName",
  constantName: "INPUT_NAME"
}
```

This makes it easy to access different variants of inputName in the templates.